examples/msi-renamed-to-authorization/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
+/node_modules/
```

Comment:
This file might not be required - I think both exclusions are covered in the parent? (same applies to other examples)

Reply:
This is copied and pasted from other examples :) We can make a pass through these later if you feel strongly about them?